python/neuroglancer/pipeline/storage.py

```diff
@@ -182,8 +182,8 @@ def _uncompress(content):
         with gzip.GzipFile(mode='rb', fileobj=stringio) as gfile:
             return gfile.read()
 
-    def list_files(self, prefix=""):
-        for f in self._interface.list_files(prefix):
+    def list_files(self, prefix="", flat=False):
```

Comment:
Hmm, I'm not sure whether it's necessary or not. We'll need some experiments. The listings returned by boto and google.cloud return iterators, so they might handle that internally.